packages/eslint-config-godaddy-react-flow/package.json

```diff
@@ -30,10 +30,11 @@
     "eslint": ">=6.0.0"
   },
   "dependencies": {
-    "babel-eslint": "^10.0.3",
+    "@babel/core": "^7.16.5",
```

Comment:
Same feedback as elsewhere; we should probably not have `@babel/core` as a direct dependency because then eslint and the app may have conflicting copies installed.